dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/association/jdbc/JdbcOrgUnitAssociationsStore.java

```diff
@@ -78,53 +79,35 @@ public SetValuedMap<String, String> getOrganisationUnitsAssociationsForCurrentUs
             } );
     }
 
-    public SetValuedMap<String, String> getOrganisationUnitsAssociations( Set<String> uids )
+    public boolean checkOrganisationUnitsAssociations( String program, String orgUnit )
```

Comment:
since this returns a boolean, wouldn't it be better to name it something like `isProgramAssociatedToOrgUnit?`